jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PluginConfigurationProcessor.java

```diff
@@ -38,9 +37,6 @@
 /** Configures and provides builders for the image building tasks. */
 class PluginConfigurationProcessor {
 
-  /** {@code User-Agent} header suffix to send to the registry. */
-  private static final String USER_AGENT_SUFFIX = "jib-gradle-plugin";
```

Comment:
I would favor this constant over having the tool name be deep within the `processCommonConfiguration` method.